x-pack/plugins/cloud_security_posture/public/pages/rules/rules_flyout.tsx

```diff
@@ -152,7 +91,7 @@ const RuleOverviewTab = ({ rule, toggleRule }: { rule: RuleSavedObject; toggleRu
       </span>
     </EuiFlexItem>
     <EuiFlexItem>
-      <EuiDescriptionList compressed={false} listItems={getOverviewCard(rule)} />
+      <EuiDescriptionList listItems={getRuleList(rule.attributes)} />
```

Comment:
is there a difference now that this one doesn't have `compressed=false` ?

Reply:
no, `false` is the default